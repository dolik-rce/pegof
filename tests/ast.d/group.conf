input ast.d/group.peg
ast
