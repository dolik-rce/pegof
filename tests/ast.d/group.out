GRAMMAR
  RULE X
    ALTERNATION
      SEQ
        TERM
          GROUP
            ALTERNATION
              SEQ
                TERM
                  STRING X
                TERM
                  STRING Y
